.github/workflows/release.yml

```diff
@@ -36,8 +36,14 @@ jobs:
           corepack enable
           pnpm --version
 
-      - name: Install dependencies
-        run: pnpm install
+      - name: Cache pnpm modules
```

Comment:
I think we can omit caching here as only lockfile is updated in this job. Plus the release workflow isn't run frequently so caches can be useless.